runtime/tools/python/ttrt/runtime/module.cpp

```diff
@@ -87,6 +89,19 @@ PYBIND11_MODULE(_C, m) {
         py::arg("executable"), py::arg("program_index"), py::arg("inputs"),
         py::arg("outputs"), "Submit a binary for execution");
   m.def("wait", &tt::runtime::wait, py::arg("event"));
+  m.def(
+      "get_op_output_tensor",
+      [](tt::runtime::OpContext &opContext,
+         tt::runtime::CallbackContext &programContext) {
+        tt::runtime::Tensor tensor =
+            tt::runtime::getOpOutputTensor(opContext, programContext);
+        return std::vector<float>(static_cast<float *>(tensor.data.get()),
+                                  static_cast<float *>(tensor.data.get()) +
+                                      tensor.volume);
```

Comment:
I think `tensor.data` probably should have been made private on the struct, this isn't really a safe thing to do.  I think if we're going to go with this approach we should remove `volume` from Tensor and have a method on it `std::vector<float> getData() const;` or something.  Then the backend runtime needs to implement this.

Reply:
fixed.